lavasnek_rs.pyi

```diff
@@ -43,7 +43,7 @@ class Node:
     now_playing: t.Optional[TrackQueue]
     is_paused: bool
     queue: t.List[TrackQueue]
-    async def get_data(self): ...
+    async def get_data(self) -> t.Dict[t.Any]: ...
```

Comment:
You can give anything to data, not just a dict, get gives an empty dict if you didn't set data to it, but if you set data to for example, a custom class, it will have that class instead.
```suggestion
    async def get_data(self) -> t.Union[t.Any, t.Dict]: ...
```

Reply:
I see, good point.

Reply:
Added the dict key/value types as `Any` here, and for set_data.